events/schema.md

```diff
@@ -4,30 +4,168 @@ Events published to the bus have a standard schema that allows SDKs and services
 
 The schema is based on JSON (and can be modelled with JSON Schema if necessary).
```

Comment:
(Unrelated, but since there's quite a bit of rethinking happening here) thought: should we follow the API's RELAX NG setup for events? JSON is definitely simpler, and as long as inline content isn't passed around (thinking article titles might appear in `data`) it's not a problem. But, it would provide the same extensibility model and (strong) validation.

Reply:
IIRC the original bus events in Continuum didn't contain any `data` apart from identifiers not to duplicate what is provided by the API; now we are introducing `data` and possibly a schema for it, which veers towards Domain Events/CQRS/Event Sourcing (but not necessarily persisting events in the long term, which simplifies schema changes). My main concern is how easy an event should be to produce/consume for a random service.